main.js

```diff
@@ -545,4 +570,10 @@ let end = false;
             process.exit(-1);
         }
     }
-})();
\ No newline at end of file
+})();
+
+//TODO
+// MD5 - Après 5 slides identiques -> erreur : arret de l'execution
+// PageRange - Debut de capture à la nieme slide et fin à la m ieme slide 
+
+// Push sur repo makina + modifier reference repos dans package.json
```

Comment:
Probably better to put that in an external file "TODO.txt", in english (and revmove last point). Or to remove it completely :)